package.json

```diff
@@ -63,6 +64,7 @@
     "@types/node": "^18.6.2",
     "@typescript-eslint/eslint-plugin": "^5.31.0",
     "@typescript-eslint/parser": "^5.31.0",
+    "babel-jest": "^28.1.3",
```

Comment:
If we can get away with not adding babel, I'd appreciate it.  We're using tsc to do transforms everywhere else.